btax/calc_final_outputs.py

```diff
@@ -105,47 +105,24 @@ def industry_calcs(params, fixed_assets, output_by_asset):
       copy=True, indicator=False)
 
 	# create weighted averages by industry/tax treatment
-	by_industry = pd.DataFrame({'delta' : by_industry_asset.groupby( ["bea_ind_code"] ).apply(wavg, "delta", "assets")}).reset_index()
+	by_industry = pd.DataFrame({'delta' : by_industry_asset.groupby( ['bea_ind_code'] ).apply(wavg, "delta", "assets")}).reset_index()
```

Comment:
These weighted averages by groups are the one thing outstanding before this PR is ready to merge. I've exhausted what I can find on SO and tried a suggestion from @PeterDSteinberg, to no avail.  I can save the by_industry_asset data frame to a csv file and do the calculations I want in Excel, so I think this should work, but I'm clearly doing something wrong with the groupby here.  cc @talumbau , @PeterDSteinberg